src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectSystems/VsMSBuildProjectSystem.cs

```diff
@@ -270,7 +246,7 @@ public void AddExistingFile(string path)
 
             if (!File.Exists(fullPath))
             {
-                throw new ArgumentNullException(String.Format(CultureInfo.CurrentCulture, Strings.PathToExistingFileNotPresent, fullPath, ProjectName));
+                throw new ArgumentNullException(string.Format(CultureInfo.CurrentCulture, Strings.PathToExistingFileNotPresent, fullPath, ProjectName));
```

Comment:
Seems odd, no doubt. Although I'm not sure what protocol expectations are made by its callers. So I'd prefer not touching this.